__main__.py

```diff
@@ -0,0 +1,59 @@
+"""
+This is the main entry point for SlateDB's Pulumi configuration. It manages
+memberships and permissions for SlateDB's open source resources such as
+Github, Discord, and Crates.io.
+"""
+
+from dataclasses import dataclass
+
+from resources import crates, discord, github
+
+@dataclass
+class Maintainer:
+    name: str
+    github: str
+    discord: str
+
+
+MAINTAINERS = [
+    Maintainer(
+        name="Chris Riccomini",
+        github="criccomini",
+        discord="839536195603922974",
+    ),
```

Comment:
Ya, this is exactly how I was thinking about it. But I deliberately made it as dead-simply/dumb as possible. Part of this was just to KISS, and part of it was because we don't have a full Python build system (uv, pdm, poetry). I just erred on the side of: what is the dumbest thing I can do?